### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by alisonmonteiro**
@dorianneto 

I've made some improvements on README.md:
1. Added a table of contents;
2. Some comments were improved;

Hope you like it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,33 +5,42 @@
...
+- [API](#api)
+- [License](#license)
+
+### Highlights
 
 * Simple API
 * Fully documented
 * Fully unit tested
 * Search for podcasts by Term
+* Serach podcasts on multiple APIs
```
Comment (dorianneto) [inline README.md]: *Search

File: README.md [hunk 2]
```diff
@@ -43,16 +52,16 @@ use PodcastCrawler\PodcastCrawler;
...
 
 $getFeed = $PodcastCrawler->find(381816509);
-var_dump($getFeed); // return array with podcast details and mp3 files
+var_dump($getFeed); // Returns an array with the podcast's detail and episodes with its mp3 files.
 ```
 
-## API
+### API
 
 See the full API through this [link](http://api.podcastcrawler.com/v0.15.1-beta/index.html).
```
Comment (dorianneto) [inline README.md]: Link to v1 API: http://api.podcastcrawler.com/v1/

File: README.md (hunk 1 above)
Comment (alisonmonteiro) [reply README.md]: Yeap. Typo. My bad.

File: README.md (hunk 2 above)
Comment (alisonmonteiro) [reply README.md]: Thank you. I'll update.

**Comment by alisonmonteiro** [comment]
@dorianneto updated.

File: README.md (hunk 1 above)
Comment (dorianneto) [reply README.md]: No problem ;)

File: README.md (hunk 2 above)
Comment (dorianneto) [reply README.md]: Nice!

**Pull request merged**